stdlib/ballerina-builtin/src/main/ballerina/auth/config_auth_store_provider.bal

```diff
@@ -21,9 +21,9 @@ import ballerina/runtime;
 @final string CONFIG_USER_SECTION = "b7a.users";
 
 documentation {
-    Represents the ballerina.conf based auth provider
+    Represents the ballerina.conf based auth store provider
```

Comment:
Don't refer to the configuration file as ballerina.conf since it's just the name given for the default config file.

Reply:
Can just simply say "Represents the configuration file based auth store provider"